Makefile.in

```diff
@@ -884,7 +884,8 @@ stamps/build-pk64: $(PK_SRCDIR) $(PK_SRC_GIT) stamps/build-gcc-newlib-stage2
 	cd $(notdir $@) && $</configure \
 		--prefix=$(INSTALL_DIR) \
 		--host=$(NEWLIB_TUPLE) \
-		--with-arch=rv64gc
+		--with-arch=rv64gc \
+		--eith-abi=lp64d
```

Comment:
`--with-abi=lp64d`?

Reply:
Hi @kito-cheng - apologies for that - thanks for spotting it!
Resolved now.
Just to clarify - the ABI should be explicitly specified otherwise it's assuming that the toolchain used was configured for `lp64d` either explicitly using `configure --with-abi=lp64d` or using the default `arch/abi` of `rv64gc/lp64d`.
But the main point of this PR is to fix the `arch/abi` used to build `pk32`.